lib/cli.js

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env node
-// import { argv } from 'yargs';
-import sgcPrompt from './sgcPrompt';
+import promptConfig from './promptConfig';
```

Comment:
How about using models? It is recommended by the [airbnb styleguide](https://github.com/airbnb/javascript#modules--use-them).

`import { config, ... } from './promptConfig'`